src/platform/web/assets/config.json

```diff
@@ -5,5 +5,13 @@
     "applicationServerKey": "BC-gpSdVHEXhvHSHS0AzzWrQoukv2BE7KzpoPO_FfPacqOo3l1pdqz7rSgmB04pZCWaHPz7XRe6fjLaC-WPDopM"
   },
   "defaultHomeServer": "matrix.org",
-  "bugReportEndpointUrl": "https://element.io/bugreports/submit"
+  "bugReportEndpointUrl": "https://element.io/bugreports/submit",
+  "oidc": {
```

Comment:
We'll eventually need a way to not hardcode this. Ideally passing it in to the client constructor.

Reply:
This is what i done here moved the hardcoded client config to main config.json file which is being passed to client. These are here so we can test the sdk.

Thirdroom have its own configration which is passed to sdk constructor.